ibmcloud/terraform/cluster/scripts/keygen.sh

```diff
@@ -68,7 +68,7 @@ for host in "${hosts[@]}"; do
         trap stop 0
 
         ssh "${opts[@]}" -o ExitOnForwardFailure=yes -S "$ssh_ctl_sock" -L "$port:$host:22" -M -f -N -l root "$bastion"
-        ssh-copy-id "${opts[@]}" -i "$key.pub" -p "$port" root@localhost
+        ssh-copy-id -f "${opts[@]}" -i "$key.pub" -p "$port" root@localhost
```

Comment:
Changing the keygen.sh script is outside of the scope of the issue this PR is fixing, so we should be making this change under a separate PR and issue

Reply:
Thanks. I have reverted this change.